### PR TITLE
Updated the Nordic (prev. Norwegian) list's metadata

### PR DESCRIPTION
**Pull request by DandelionSprout**
There's been some updates in my Norwegian list, that I feel could need to be reflected in the various desktop/app/extension versions of AdGuard.

* In December 2018, after realising that Schack had most likely ceased work on his Danish adblock list, and that roughly a quarter of his list's entries had ceased to work or were for defunct domains, I decided to expand my list to also include Danish sites. While I was at it, I wrote Icelandic entries as well (initially as a self-challenge to see how much I could shrink the *Icelandic ABP List* list), and a handful of Faroese ones. Therefore the list's name is now officially **Dandelion Sprout's Nordic filters** (for tidier websites).
* * However, it does *not* cover (non-Sami) Finnish or Swedish websites, as the currently maintained lists for those languages are superior to what I could've cooked up for them.
* After I discovered that the `! Description:` tag was a thing that existed in AdGuard's desktop version, I ended up trying to use it on the Nordic list (and on ≥30 of my other and less frequently used lists as well). However, now that I see that it did not automatically apply to lists that are subscribed to from this repo, I'm adding it manually here.
* Slowly and over time, my Nordic list has also gained a secondary focus on trying to prevent mass scams and phishing sites, especially ones that so many people fell for that they got articles about them on Norwegian newssites or consumer boards. Therefore I've decided to add `purpose:security` to the metadata as well.

If you have any questions about all this, feel free to ask me about them.

### CODE REVIEW

**Comment by ameshkov** [comment]
@Alex-302  I just realized that we don't have any filter marked as "recommended" for `lang:no`.

@DandelionSprout do you mind if we mark your filter as recommended? It will be enabled by default for the users who speak Norwegian.

**Comment by DandelionSprout** [comment]
I'm surprised it wasn't already recommended to those users, so go ahead.

File: filters/ThirdParty/filter_249_NorwegianList/metadata.json [hunk 1]
```diff
@@ -11,6 +11,9 @@
   "tags": [
     "purpose:ads",
     "purpose:security",
-    "lang:no"
+    "lang:no",
+    "lang:da",
+    "lang:is",
+    "lang:fo"
```
Comment (ameshkov) [inline filters/ThirdParty/filter_249_NorwegianList/metadata.json]: This one needs to be added to https://github.com/AdguardTeam/FiltersRegistry/blob/master/tags/metadata.json

Comment (ameshkov) [reply filters/ThirdParty/filter_249_NorwegianList/metadata.json]: Also, if you don't mind, add "recommended" tag here as well.

Comment (DandelionSprout) [reply filters/ThirdParty/filter_249_NorwegianList/metadata.json]: Presuming that you want me to take care of that, I'll get around to it right away. 🙂

Comment (ameshkov) [reply filters/ThirdParty/filter_249_NorwegianList/metadata.json]: Also, every tag has a name and description: https://github.com/AdguardTeam/FiltersRegistry/blob/master/locales/en/tags.json

**Comment by DandelionSprout** [comment]
Is it okay if I add `"recommended"` to Frellwit's Swedish list as well? His list is quite stellar, and I look up to him for his filterlist writing skills.

**Comment by DandelionSprout** [comment]
And, now that I've added `list:fo` as tagId 63, is there some other file where I/we now have to create an `"tagId": 63 entry in?

**Comment by ameshkov** [comment]
> Is it okay if I add `"recommended"` to Frellwit's Swedish list as well? His list is quite stellar, and I look up to him for his filterlist writing skills.

Well, no problem if you say so.

> And, now that I've added `list:fo` as tagId 63, is there some other file where I/we now have to create an `"tagId": 63 entry in?

One last thing would be to add it's English name and description here:
https://github.com/AdguardTeam/FiltersRegistry/blob/master/locales/en/tags.json

**Comment by ameshkov** [comment]
Looks good, thank you!

Now we just need to wait for @Alex-302 and for Travis builds to complete

**Comment by DandelionSprout** [comment]
Seeing that the Travis build failed, I looked around on the metadata of other locale lists, and saw that the tag system worked differently than I thought it did.

So I guess I'll have to fix that too.

**Comment by ameshkov** [comment]
Uh, no, it was fine in the previous commit.

The build has failed due to another filter being unavailable at that moment.

**Comment by DandelionSprout** [comment]
Aaaaah. I was just slightly confused by how the metadata for AdGuard's own list seemed at first sight to use a tiered tree, which it didn't.

I'll correct it right away.

**Comment by DandelionSprout** [comment]
This is the first time I've made first-hand contributions to this repo, so I've had to try to figure out the JSON files' functionalities in a hurry, a good ol' "baptism by fire". Hopefully that's understandable.

**Comment by ameshkov** [comment]
Thank you! Everything is good now, let's wait. I'll restart the Travis build if it fails again.

The commit history is quite a story now:)

**Pull request merged**